src/shogun2-util/src/main/java/de/terrestris/shogun2/util/http/HttpUtil.java

```diff
@@ -486,6 +488,138 @@ private static Response postParams(HttpPost httpRequest,
 	}
 
 	/**
+	 * Perform HTTP PUT with empty body
+	 *
+	 * @param uri
+	 * @return
+	 * @throws HttpException
+	 * @throws URISyntaxException
+	 */
+	public static Response put(URI uri) throws URISyntaxException, HttpException {
+		return putBody(new HttpPut(uri), StringUtils.EMPTY, ContentType.APPLICATION_JSON, null, null);
```

Comment:
Why do we set `ContentType.APPLICATION_JSON` here (and in the ones below) without having a body content?